src/v/cloud_storage_clients/client_pool.cc

```diff
@@ -19,10 +19,13 @@
 #include <seastar/core/smp.hh>
 
 #include <algorithm>
+#include <chrono>
 #include <optional>
 #include <random>
 #include <utility>
 
+using namespace std::chrono_literals;
```

Comment:
Some of this feels unrelated or maybe another commit?

Reply:
it's related: by removing headers during the refactoring in `net::` we unearthed places that were getting their chrono_literals through that path.

Reply:
unless you're saying that those chrono literals could have been added into a previous commit?

Reply:
Oh I see, no it's fine.